manager/job/gc.go

```diff
@@ -74,3 +74,31 @@ func (gc *gc) Serve() {
 func (gc *gc) Stop() {
 	close(gc.done)
 }
+
+// deleteInBatches deletes jobs in batches.
+func (gc *gc) deleteInBatches(ctx context.Context) error {
```

Comment:
Could we delete multiple items at once by specifying a limit, rather than executing a query before deletion, which increases costs?